x/leveragelp/keeper/begin_blocker.go

```diff
@@ -67,15 +80,15 @@ func (k Keeper) LiquidatePositionIfUnhealthy(ctx sdk.Context, position *types.Po
 	h, err := k.GetPositionHealth(ctx, *position)
 	if err != nil {
 		ctx.Logger().Error(errors.Wrap(err, fmt.Sprintf("error updating position health: %s", position.String())).Error())
-		return false, true, sdk.ZeroDec(), err
+		return false, false, sdk.ZeroDec(), err
```

Comment:
these conditions are used in `closePositons`, could you please verify changing this to false doesn't impact that part ?

Reply:
No, it doesn't. Instead of returning the variable name `earlyReturn`, I changed it to `closeAttempted`. 
`earlyReturn` was ambiguous - it wasn't very clear from where it's returning.